policy/HscMapper.paf

```diff
@@ -718,6 +718,23 @@ datasets: {
         tables:        raw
         tables:        raw_skyTile
     }
+    deep_safeClipAssembleCoadd_config: {
+        template:      "config/deep_assembleCoadd.py"
+        python:        "lsst.pipe.tasks.assembleCoadd.SafeClipAssembleCoaddConfig"
+        persistable:      "Config"
+        storage:    "ConfigStorage"
```

Comment:
I realise wonky indenting is endemic in this file, but please don't add more of it!